docs/ccdproc/reduction_toolbox.rst

```diff
@@ -355,6 +355,14 @@ operations in `ccdproc`.
     # Save the mask as "mask" attribute of the ccd
     ccd.mask = mask
 
+Another method for creating a mask is using the `~ccdproc.ccdmask` task.  This
+task will produced a data aray where good pixels have a value of zero and bad
+pixels have a value of one.   This task follows the same algorithm used in the
+iraf ccdmask task. 
+
+     >>> ccd.mask =  ccdmask(ccd, ncmed=7, nlmed=7, ncsig=15, nlsig=15, lsigma=9, 
+     ...                     lsigma=9, hsigma=9, ngood=5)
```

Comment:
`lsigma` is repeated: at the end of the first line and the beginning of the second